packages/angular/ssr/schematics/ng-add/index.ts

```diff
@@ -254,7 +303,7 @@ function routingInitialNavigationRule(options: UniversalOptions): Rule {
 }
 
 function addDependencies(): Rule {
-  return (_host: Tree) => {
+  return () => {
```

Comment:
nit: can remove this and just return the chain directly if the tree parameter isn't used